api/rbdigital.py

```diff
@@ -1608,9 +1608,22 @@ def isbn_info_to_metadata(cls, book, include_bibliographic=True, include_formats
             Identifier.RB_DIGITAL_ID, rbdigital_id
         )
 
+        # medium is both bibliographic and format information.
+
+        # options are: "eBook", "eAudio"
+        rbdigital_medium = book.get('mediaType', None)
+        if rbdigital_medium and rbdigital_medium not in cls.rbdigital_medium_to_simplified_medium:
+            cls.log.error(
+                "Could not process medium %s for %s", rbdigital_medium, rbdigital_id)
+
+        medium = cls.rbdigital_medium_to_simplified_medium.get(
+            rbdigital_medium, Edition.BOOK_MEDIUM
+        )
+
```

Comment:
I moved this code out of the `if include_bibliographic` block so it would always run and we would never have a situation where it was unclear what medium a title was. The medium is bibliographic information, but the RBdigital code also uses it to determine format information.

We could probably get rid of the `include_bibliographic` thing altogether, but there's no rush.

Reply:
I checked the other license sources and didn't find any other cases where we relied on the default Metadata.medium behavior that was removed from core.